app/assets/javascripts/catalog_manager/update.js.coffee

```diff
@@ -96,6 +96,34 @@ $(document).ready ->
   $('.service_rate').live('blur', ->
     $(this).formatCurrency()
   )
+
```

Comment:
You can attach the same listener to multiple elements using commas like:
`$('[display="Federal Rate"], [display="Corporate Rate"], [display="Other Rate"], etc.').live(...)`